src/framework/mlt_consumer.c

```diff
@@ -852,8 +852,10 @@ static void *consumer_read_ahead_thread( void *arg )
 		pthread_cond_broadcast( &priv->queue_cond );
 		pthread_mutex_unlock( &priv->queue_mutex );
 
+		mlt_log_timings_begin();
 		// Get the next frame
 		frame = mlt_consumer_get_frame( self );
+		mlt_log_timings_end(NULL, "mlt_consumer_get_frame");
```

Comment:
> Should the service be provided instead of NULL?

In general, yes, but I suppose it can create some reading clutter. Before I merge, I just want to get @max-verem feedback about using mlt_service instead of NULL in calls to mlt_log_timings(). Otherwise, I am fine with the rest of the pull request.